userspace/libpman/src/lifecycle.c

```diff
@@ -0,0 +1,45 @@
+#include "state.h"
+
+int libpman__open_probe()
+{
+	g_state.skel = bpf_probe__open();
+	if(!g_state.skel)
+	{
+		libpman__print_error("failed to open BPF skeleton");
```

Comment:
Similarly here: maybe a `get_last_error` library function might be a better option here. This is pretty much the same pattern we have in libscap.

Reply:
Agree: changing in the future is not a big deal; i'd rather keep the thing as it is!